README.md

```diff
@@ -152,7 +152,7 @@ Run the tests:
 
 Run the tests with specific versions of GOLANG and PULSAR:
 
-    make test GOLANG_VERSION=1.20 PULSAR_VERSION=2.10.0
+    make test GO_VERSION=1.20 PULSAR_VERSION=2.10.0
```

Comment:
Why change this name?

Reply:
Did not match with the name of the variable inside the Makefile.
![image](https://github.com/apache/pulsar-client-go/assets/30929349/6c163a1f-110c-41cd-8713-824e8dedc663)